clouddriver-ecs/src/integration/resources/testoperations/createServerGroup-spelProcessedArtifact-EC2-targetGroupMappings.json

```diff
@@ -0,0 +1,99 @@
+{
+  "account": "ecs-account",
+  "application": "ecs",
+  "availabilityZones": {
+    "us-west-2": [
+      "us-west-2a",
+      "us-west-2c"
+    ]
+  },
+  "capacity": {
+    "desired": 1,
+    "max": 1,
+    "min": 1
+  },
+  "cloudProvider": "ecs",
+  "computeUnits": 256,
+  "credentials": "ecs-account",
+  "dockerImageAddress": "nginx",
+  "ecsClusterName": "integSpELProcessedArtifactEC2TgMappings-cluster",
+  "launchType": "EC2",
+  "networkMode": "bridge",
+  "placementStrategySequence": [],
+  "reservedMemory": 512,
+  "stack": "integSpELProcessedArtifactsEC2TgMappingsStack",
+  "freeFormDetails" : "detailTest",
+  "targetGroupMappings": [
+    {
+      "containerName": "application",
+      "containerPort": 80,
+      "targetGroup": "integSpELProcessedArtifactEC2TgMappings-targetGroup"
+    }
+  ],
+  "useTaskDefinitionArtifact" : true,
+  "evaluateTaskDefinitionArtifactExpressions" : true,
+  "taskDefinitionArtifactAccount"  :  "my-github",
+  "source" : {
+    "account" :  "ecs-account",
+    "region" :  "us-west-2",
+    "asgName" :  "ecs",
+    "useSourceCapacity" : true
+  },
+  "resolvedTaskDefinitionArtifact": {
+    "account": "ecs-account",
+    "type": "ecs",
+    "customKind" :  true,
+    "name" : "application",
+
+    "location" : "us-west-2",
+    "reference" : "refernce",
+    "metadata" : null,
+    "artifactAccount" : "my-github",
+    "provenance" : "prov",
+    "uuid" : "uid-123"
+  },
+  "spelProcessedTaskDefinitionArtifact" : [{
+  "family": "PLACEHOLDER",
```

Comment:
please indent the contents of the artifact so it appears as a child of `spelProcessedTaskDefinitionArtifact` (they're on the same vertical line right now)

Reply:
oops, fixed.